src/test/java/com/stripe/model/PagingIteratorTest.java

```diff
@@ -44,10 +46,26 @@ public String getId() {
 
   private static class PageableModelCollection extends StripeCollection<PageableModel> {}
 
+  private static class ReferencesPageableModel extends ApiResource implements HasId {
+    @Getter String id;
+
+    @Getter PageableModelCollection pages;
+
+    public static ReferencesPageableModel retrieve(RequestOptions options) throws StripeException {
+      return request(
+          ApiResource.RequestMethod.GET,
+          classUrl(ReferencesPageableModel.class),
+          new HashMap<String, Object>(),
+          ReferencesPageableModel.class,
+          options);
+    }
+  }
+
+  List<String> pages;
```

Comment:
Is this new field used outside of setUpMockPages ?

Reply:
Ah good catch.